src/main/java/com/academia/library/model/Book.java

```diff
@@ -0,0 +1,65 @@
+package com.academia.library.model;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToMany;
+import javax.persistence.ManyToOne;
+import javax.persistence.SequenceGenerator;
+import javax.persistence.Table;
+import java.time.LocalDate;
+import java.util.List;
+
+@Getter
+@Setter
+@EqualsAndHashCode
+@ToString
+@NoArgsConstructor
+@Entity
+@Table(name = "books")
+public class Book {
+
+    @Id
+    @SequenceGenerator(name = "book_id_seq", sequenceName = "book_id_seq", allocationSize = 1, initialValue = 100)
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "book_id_seq")
+    private Long id;
+
+    @Column
+    private Double price;
+
+    @Column
+    private String title;
+
+    @Column
+    private String imageUrl;
+
+    @Column
+    private LocalDate createAt;
+
+    @Column
+    private LocalDate updateAt;
+
+    @ToString.Exclude
+    @EqualsAndHashCode.Exclude
+    @ManyToOne
+    @JoinColumn(name="author_id", nullable=false)
+    private Author author;
+
+    @ToString.Exclude
+    @EqualsAndHashCode.Exclude
+    @ManyToOne
+    @JoinColumn(name="tag_id", nullable=false)
+    private Tag tag;
+
+    @ManyToMany(mappedBy = "books")
+    private List<Order> orders;
+}
```

Comment:
As I recommended you latter please use commands CTRL+ALT+ L/O

Reply:
Ok